utils/graphql-schemas/schema-docs.md

```diff
@@ -0,0 +1,2306 @@
+
+---
+**NR Internal** | [#help-nr1-dev-experience](https://newrelic.slack.com/archives/CPE597DNY) | visibility(customer)
+
+
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>id</strong></td>
+<td valign="top">ID!</td>
+<td>
+
+The unique identifier for the install plan step
+
+---
+**NR Internal** | [#help-nr1-dev-experience](https://newrelic.slack.com/archives/CPE597DNY) | visibility(customer)
+
+
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>primary</strong></td>
+<td valign="top"><a href="#nr1cataloginstallplandirective">Nr1CatalogInstallPlanDirective</a>!</td>
+<td>
+
+Provides context about how the primary install plan step should proceed
+
+---
+**NR Internal** | [#help-nr1-dev-experience](https://newrelic.slack.com/archives/CPE597DNY) | visibility(customer)
+
+
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>target</strong></td>
+<td valign="top"><a href="#nr1cataloginstallplantarget">Nr1CatalogInstallPlanTarget</a>!</td>
+<td>
+
+Provides context about where the install will occur
+
+---
+**NR Internal** | [#help-nr1-dev-experience](https://newrelic.slack.com/archives/CPE597DNY) | visibility(customer)
+
+
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### Nr1CatalogInstallPlanTarget
+
+Represents the location of an install
+
+---
+**NR Internal** | [#help-nr1-dev-experience](https://newrelic.slack.com/archives/CPE597DNY) | visibility(customer)
+
+
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>destination</strong></td>
+<td valign="top"><a href="#nr1cataloginstallplandestination">Nr1CatalogInstallPlanDestination</a>!</td>
+<td>
+
+Provides context on the location the install will take place
+
+---
+**NR Internal** | [#help-nr1-dev-experience](https://newrelic.slack.com/archives/CPE597DNY) | visibility(customer)
+
+
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>os</strong></td>
+<td valign="top">[<a href="#nr1cataloginstallplanoperatingsystem">Nr1CatalogInstallPlanOperatingSystem</a>!]!</td>
+<td>
+
+Provides context for the operating system that will be targeted
+
+---
+**NR Internal** | [#help-nr1-dev-experience](https://newrelic.slack.com/archives/CPE597DNY) | visibility(customer)
+
+
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>type</strong></td>
+<td valign="top"><a href="#nr1cataloginstallplantargettype">Nr1CatalogInstallPlanTargetType</a>!</td>
+<td>
+
+Provides context for the type of installation that will take place
+
+---
+**NR Internal** | [#help-nr1-dev-experience](https://newrelic.slack.com/archives/CPE597DNY) | visibility(customer)
+
+
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### Nr1CatalogQuickstart
+
+Information about the quickstart
+
+---
+**NR Internal** | [#help-nr1-dev-experience](https://newrelic.slack.com/archives/CPE597DNY) | visibility(customer)
```

Comment:
We may not want to expose our slack channels and field visibility publicly

Reply:
One idea to make this easy is to create a "regular" account which is non NR admin, create a key from there, and use that key to generate these docs. NerdGraph will strip this out for you in that case so that you don't need to worry about creating regexes to strip this out yourself.